Src/Base/AMReX_GpuQualifiers.H

```diff
@@ -8,6 +8,12 @@
 #include <hip/hip_runtime.h>
 #endif
 
+#if defined(AMREX_USE_CUDA) && defined(AMREX_CXX_PGI)
```

Comment:
Does this work?
```suggestion
#if defined(AMREX_USE_CUDA) && (defined(AMREX_CXX_PGI) || defined(AMREX_CXX_NVHPC))
```
In case, in the future nvc++ no longer defines `__PGI`.